sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/GenerateMetadataMojo.java

```diff
@@ -19,77 +19,81 @@
  */
 package org.sonar.updatecenter.mojo;
 
+import java.io.File;
+import java.io.IOException;
 import org.apache.maven.plugin.AbstractMojo;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.sonar.updatecenter.common.UpdateCenter;
+import org.sonar.updatecenter.mojo.editions.EditionTemplatesLoaderImpl;
+import org.sonar.updatecenter.mojo.editions.EditionsGenerator;
 
-import java.io.File;
-
-/**
- * @goal generate-metadata
- * @requiresProject false
- */
+@Mojo(name = "generate-metadata", threadSafe = true, requiresProject = false)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder annotation arguments to match the order of declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3340)